bigtable/tests/unit/test_row.py

```diff
@@ -367,6 +367,27 @@ def test_commit(self):
         )])
         self.assertEqual(row._pb_mutations, [])
 
+    def test_retry_commit_exception(self):
+        import threading
+        import grpc._channel
+        from google.cloud.bigtable.row import _retry_commit_exception
+
+        class State:
+            condition = threading.Condition()
+            code = grpc.StatusCode.UNAVAILABLE
+            details = 'Endpoint read failed'
+
+        def _check_rendezvous_exception(exc):
+            return _retry_commit_exception(exc)
+
+        expected_result = True
+        result = _check_rendezvous_exception(
+            grpc._channel._Rendezvous(State, None, None, 0))
```

Comment:
At no point should you be using private members in grpc. Create a mock using `grpc.Call/grpc.RpcError` as the spec, see https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/api_core/tests/unit/test_exceptions.py#L188